instance/tests/integration/integration_instance.py

```diff
@@ -42,6 +43,9 @@
 
 # Tests #######################################################################
 
+@override_settings(
+    DEFAULT_LMS_PREVIEW_DOMAIN_PREFIX='integ-',
```

Comment:
Thanks for clarifying, @mtyaka!

> It should be self.instance.domain instead of self.instance.lms_preview_domain in https://github.com/open-craft/opencraft/blob/master/instance/models/openedx_appserver.py#L290

@bdero Would you be OK to make that change as part of this PR?